core/src/integrationTest/kotlin/ch/ergon/dope/helper/DBProvider.kt

```diff
@@ -0,0 +1,68 @@
+package ch.ergon.dope.helper
+
+import com.couchbase.client.kotlin.Cluster
+import kotlinx.coroutines.runBlocking
+import org.testcontainers.couchbase.BucketDefinition
+import org.testcontainers.couchbase.CouchbaseContainer
+import org.testcontainers.utility.DockerImageName
+import kotlin.time.Duration.Companion.seconds
+
+object DBProvider {
+    private val container = CouchbaseContainer(
+        DockerImageName.parse("couchbase/server:community-7.6.2"),
+    )
+    var cluster: Cluster
+
+    init {
+        container.withBucket(BucketDefinition("testBucket"))
+        container.start()
```

Comment:
would put these two methods into a _initContainer_-method and the following statement into a _initCluster_-method

Reply:
and then maybe rename _fillDatabase_ to _initDatabase_

Reply:
I extracted the container initialization into `initContainer`, but left the cluster in the `init` method, so the cluster val doesn't have to be a `var`, also renamed fillDatabase to initDatabase